Firebase/Core/FIRHeartbeatInfo.m

```diff
@@ -16,6 +16,7 @@
 #import <GoogleUtilities/GULHeartbeatDateStorage.h>
 #import <GoogleUtilities/GULLogger.h>
 
+static long secondsInDay = 84000;
```

Comment:
nit: This variable should be declared `const`. Also, there are 86400 seconds in one day, not 84000.

Reply:
Thanks a lot!